api/user_test.go

```diff
@@ -32,8 +32,9 @@
 package cvpapi
```

Comment:
In later commits, we can add assert package here too for better failure handling.

Reply:
Yep, we can add that. Asserts are always useful in testing.